src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,10 +1,45 @@
-import React from 'react';
-import Layout from 'components/Layout/Layout';
+import React from "react";
+import { useSelector } from "react-redux";
+import Layout from "components/Layout/Layout";
+import { Link } from "react-router-dom";
+import "./GetInvolved.scss";
 
 const GetInvolved = () => {
+  let clubDataAll = useSelector((state) => state.clubs);
+  let pageInfo = useSelector((state) => state.pages["join-a-club"]);
```

Comment:
Lets pass this as data from the routes component: example home page - https://github.com/Hawaian-United-Okinawa-Association/huoa-react/blob/main/src/config/routes.js